saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/search/searchFormMode.js

```diff
@@ -51,8 +53,8 @@ var SearchFormMode = {
 	 * @param callback - function invoked when hide animation ends
 	 */
 	hide: function(callback) {
-		SearchFormMode.hideButton(SearchFormMode.getButtonLess());
-
+		SearchFormMode.hideButtonLess();
+		SearchFormMode.showButton(SearchFormMode.getButtonMore());
```

Comment:
You have hideButtonLess, but showButton(buttonMore). Why not to do it coherent? For example hideButton(buttonLess), showButton(buttonMore)?